java/src/org/openqa/selenium/manager/SeleniumManager.java

```diff
@@ -147,20 +148,15 @@ private File getBinary() {
 
   /**
    * Determines the location of the correct driver.
-   * @param driverName which driver the service needs.
+   * @param options Browser Options instance.
    * @return the location of the driver.
    */
-    public String getDriverPath(String driverName) {
-        if (!ImmutableList.of("geckodriver", "chromedriver", "msedgedriver", "IEDriverServer").contains(driverName)) {
-            throw new WebDriverException("Unable to locate driver with name: " + driverName);
-        }
-
-        String driverPath = null;
+    public String getDriverPath(Capabilities options) {
         File binaryFile = getBinary();
-        if (binaryFile != null) {
-          driverPath = runCommand(binaryFile.getAbsolutePath(),
-                    "--driver", driverName.replaceAll(EXE, ""));
+        List<String> commandList = Arrays.asList(binaryFile.getAbsolutePath(), "--browser", options.getBrowserName());
```

Comment:
Just curious, why is the null check removed for the `binaryFile`?

Reply:
Not sure. Probably oversight after changing things too many times.